7.figures/nbconverted/precision_recall_curves.r

```diff
@@ -0,0 +1,65 @@
+suppressPackageStartupMessages(library(dplyr))
+suppressPackageStartupMessages(library(ggplot2))
+
+# Load figure themes and colors
+source("figure_themes.R")
+
+# Set output files
+output_file <- file.path(
+    "figures", "pr_curves_multiclass_deepprofiler.png"
```

Comment:
```suggestion
    "figures", "PR_curves_multiclass_DP.png"
```

This may remove clarity from the type of features we are looking at, but it might be worth staying consistent with the naming convention I used in the other modules. I think by the 8th module and including documentation of `CP = CellProfiler and DP=DeepProfiler` those that browse through this repo will know that what CP/DP stand for.

 This change is up to your discretion.

Reply:
sounds good! Yes, let's keep things consistent.

Reply:
addressed in df1c8019a32b258c892db2e3facd8a8aeaa59a86